src/house.h

```diff
@@ -31,6 +31,8 @@ static const HouseID NEW_HOUSE_OFFSET = 110;    ///< Offset for new houses.
 static const HouseID NUM_HOUSES       = 512;    ///< Total number of houses.
 static const HouseID INVALID_HOUSE_ID = 0xFFFF;
 
+static const int HOUSE_NUM_ACCEPTS = 16; ///< Max number of cargoes accepted by a tile
```

Comment:
Nah, in my head CargoID refers to one specific cargo, it does not count cargo types. If it was an upper limit on cargo type ID numbers it would be appropriate, but this is the size of a list of IDs.

Reply:
should probably be a uint though, as is array size